pkg/rtc/mediatracksubscriptions.go

```diff
@@ -869,9 +869,9 @@ func (t *MediaTrackSubscriptions) downTrackClosed(
 		if sender == nil {
 			return
 		}
-		t.params.Logger.Debugw("removing peerconnection track",
-			"subscriber", sub.Identity(),
-			"subscriberID", subscriberID,
+		sub.GetLogger().Infow("removing peerconnection track",
+			"publisher", subTrack.PublisherIdentity(),
+			"publisherID", subTrack.PublisherID(),
```

Comment:
Use the subscriber logger as it is confusing otherwise. This is acting on a peer connection of the subscriber.